pages/index.tsx

```diff
@@ -1,18 +1,138 @@
-import React from 'react'
+import React, { useState, useEffect, useCallback } from 'react'
 import Page from '../client/components/page'
 
 // Polaris
-import { DisplayText, Layout } from '@shopify/polaris'
+import {
+	DisplayText,
+	Layout,
+	Button,
+	Checkbox,
+	Form,
+	FormLayout,
+	TextField
+} from '@shopify/polaris'
 
 // Events
 import Events from '../client/components/events'
-import { IndexProps } from '../shared/types'
-import { fetchRawEvents, enrichEvents } from '../client/events'
+import { IndexProps, RichEventType, RichEventsType } from '../shared/types'
+import { fetchRawEvents, enrichEvents, fetchRawEventDb } from '../client/events'
 import { IncomingMessage } from 'http'
 
+import Daet from 'daet'
```

Comment:
no need for Daet here, can just use `new Date().getTime()` or `new Date().getISOString()`